docs/common/module/coffee-module-mongodb.adoc

```diff
@@ -2,102 +2,204 @@
 = coffee-module-mongodb
 
 A modul célja a MongoDB NOSQL adatbázis kezelésének támogatása, amihez különböző kiszolgáló osztályokat és metódusokat tartalmaz.
+CDI extension kiterjesztésen alapszik http://cdi-spec.org/[cdi-spec]. 
 
-Konfiguráció yaml-ben:
+== CDI Extension készítés
```

Comment:
== Hogy készült a Mongo CDI Extension
... valami lyen cím illene neki. Jó hogy ilyen részletes, de mivel ez önmagában túl mutat a coffee fejlesztői dokumentáción, annyira nem illik ide. Akár egy új HOW-TO bekezdés is lehetne... Ötleteljünk...